hack/make.sh

```diff
@@ -93,6 +93,10 @@ if [ ! "$GOPATH" ]; then
 	exit 1
 fi
 
+if [ -z "$DOCKER_EXPERIMENTAL" ]; then
```

Comment:
I know you want all those fancy new features in, but this isn't the way to do that LOL